packages/expo-dev-menu/ios/Modules/DevMenuManagerProvider.m

```diff
@@ -1,12 +0,0 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
-
-#import <React/RCTBridgeModule.h>
-
-@interface RCT_EXTERN_REMAP_MODULE(ExpoDevMenuManagerProvider, DevMenuManagerProvider, NSObject)
-
-+ (BOOL)requiresMainQueueSetup
-{
-  return true;
```

Comment:
did this get moved somewhere?

Reply:
do you mean the module? its no longer needed

Reply:
i meant specifically the `requiresMainQueueSetup` value of true -- but if this module was removed entirely (rather than having its functionality moved around) then nvm, all good